src/react/components/StyleEditor/constants.js

```diff
@@ -6,6 +6,16 @@ export const ANTD_FORMLAYOUT = {
     wrapperCol: { span: 24 } // width of wrapping column in AntD grid settings -> full width = own row inside element
 };
 
+// index passed to array corresponds straight the old value used for each fill style
+export const FILL_STYLE_FALLBACK = [
+    'thin_diagonal',
+    'thick_diagonal',
+    'thin_horizontal',
+    'thick_horizontal',
+    'transparent',
+    'solid'
```

Comment:
Import bundles/mapping/mapmodule/oskariStyle/constants.js and use the constants instead of repeating the string values here so it's more easy to find connections between these values.

Reply:
Removed these for now.